### ISSUE TITLE
Dependency on lodash.isarray

### ISSUE THREAD

**Issue by daltones** [sass/node-sass#1829]
I saw that 79e86f32ce3661569748164d26e2f2667a79699a introduced a dependency on [`lodash.isarray`](https://www.npmjs.com/package/lodash.isarray) among others.

Is that really necessary? Any particular reason to not use the standard `Array.isArray()`?

`Array.isArray()` already appears in other parts of the code and we're having this message on install:

```
warning node-sass > lodash.isarray@4.0.0: This package is deprecated. Use Array.isArray.
```

**Comment by xzyfer** [comment]
`Array.isArray` is not available until Node 4.6.2. We support back to Node 0.10.

**Comment by xzyfer** [comment]
My apologies you are correct. Fixed by #1830.
